models/intermediate/zendesk/int_rag_zendesk__ticket_document.sql

```diff
@@ -11,7 +11,7 @@ with tickets as (
 ), ticket_details as (
     select
         tickets.ticket_id,
-        tickets.url as url_reference,
+        replace(replace(cast(tickets.url as {{ dbt.type_string() }}), '/api/v2/tickets/', '/agent/tickets/'), '.json', '') as url_reference,
```

Comment:
This looks good to me, but I want to make sure this is expected within our own internal use case. I believe in our internal use case we do not include the `/agent` portion. Although I agree from your docs reference and also seeing that this does work that I would agree the inclusion of agent works here.

Let's circle up with internal folks and confirm this approach before going live.

Reply:
Confirmed internally that this new url stucture is ok to use.